tests/test_bench.py

```diff
@@ -0,0 +1,24 @@
+import os
+from kubernetes import client, config
+os.sys.path.append(os.path.dirname(os.getcwd()))
+
+from oc import pod, pod_exec
+from tests import test_radosbench as radosbench
+
+
+def run():
+    config.load_kube_config()
+    v1 = client.CoreV1Api()
+    ret = v1.list_pod_for_all_namespaces(watch=False,
```

Comment:
If the coding convention with brackets gets approved from Raz, please align here as well

Reply:
Done